src/components/current-task/CurrentTask.jsx

```diff
@@ -24,7 +24,7 @@ function CurrentTask() {
   const [, setPlay] = useContext(PlayContext);
   const [isChecked] = useContext(FullscreenContext);
```

Comment:
isChecked could be slightly more descriptive to not confuse it with a task being checked, but other than that looks good!